docs/containers/app-service.md

```diff
@@ -88,6 +89,24 @@ In the previous section, the image is pushed to a remote container registry. Now
 
     ![Web Application](images/app-service/webapp-homepage.png)
 
+## Deploy to ACI
+
+You can easily run your containers in Azure using [Azure Container Instances (ACI)](https://azure.microsoft.com/services/container-instances/). First, deploy the container to a container registry, such as Docker Hub or ACR, as described earlier in this article. Then, find the container in the **Registries** section of the Docker view. Use the **Refresh** button if you don't see it there. Right-click the entry for the image you want, and choose **Deploy image to Azure Container Instances**.
+
+![Deploy to ACI menu item](images/app-service/deploy-aci-menu.png)
+
+Choose an existing context, or use **Create new ACI Context**, and then choose the resource group. The instance is set up and started in Azure. 
```

Comment:
I like something like this for the second sentence.  "The container will be started in ACI."

Reply:
"The container is started in ACI" - ok?  We are supposed to use present tense in docs, not "will be".